### PR TITLE
(Developers) Improve Dogshell docs

### PR DESCRIPTION
**Pull request by hestonhoffman**
### What does this PR do? What is the motivation?

The initial motivation was to improve the experience for users on other data centers, but this page really needed a complete rewrite. I also renamed the file and added an alias.

I tested the steps myself with a us5 sandbox, so the instructions should be good to go.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/developers/guide/dogshell.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+aliases:
+  - /developers/faq/dogshell-quickly-use-datadog-s-api-from-terminal-shell
+  - /developers/guide/dogshell-quickly-use-datadog-s-api-from-terminal-shell
+---
+
+You can use the Datadog API on the command line using a wrapper called Dogshell.
+
+## Install Dogshell
+
+Dogshell comes with the officially supported [datadogpy Python library][1], which is often used to send data to Datadog with [DogStatsD][2]. To install the library with PIP, run the following command:
```
Comment (neko-dd) [inline content/en/developers/guide/dogshell.md:13]:
```suggestion
Dogshell comes with the officially supported [`datadogpy` Python library][1], which is often used to send data to Datadog with [`DogStatsD`][2]. To install the library with PIP, run the following command:
```

File: content/en/developers/guide/dogshell.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+{{< code-block lang="shell" >}}
+pip install datadog
+{{< /code-block >}}
+
+Depending on your environment, you might have to add the library to your PATH. See the [datadogpy GitHub repo][3] for alternate installation instructions.
```
Comment (neko-dd) [inline content/en/developers/guide/dogshell.md:19]:
```suggestion
Depending on your environment, you might have to add the library to your PATH. See the [`datadogpy` GitHub repo][3] for alternative installation instructions.
```

File: content/en/developers/guide/dogshell.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+1. Create a `.dogrc` file in your home directory:
+   {{< code-block lang="shell" >}}
+touch ~/.dogrc
+{{< /code-block >}}
+
+1. Add the following content to the file, replacing `DATADOG_API_KEY` and `YOUR_APPLICATION_KEY` with your API key and application key respectively:
+   ```conf
+   [Connection]
+   apikey = <DATADOG_API_KEY>
+   appkey = <YOUR_APPLICATION_KEY>
```
Comment (neko-dd) [inline content/en/developers/guide/dogshell.md:35]: Don't we usually use DD_API_KEY and DD_APP_KEY?

File: content/en/developers/guide/dogshell.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+   api_host = https://{{< region-param key="dd_site">}}
+   ```
+
+   <div class="alert alert-info">You can create multiple configuration files if you need to run commands against different environments. Use the <code>--config</code> flag to specify the path to an alternate configuration file.</div>
```
Comment (neko-dd) [inline content/en/developers/guide/dogshell.md:39]:
```suggestion
   <div class="alert alert-info">You can create multiple configuration files if you need to run commands against different environments. Use the <code>--config</code> flag to specify the path to an alternative configuration file.</div>
```

File: content/en/developers/guide/dogshell.md (hunk 3 above)
Comment (hestonhoffman) [reply content/en/developers/guide/dogshell.md:35]:
```suggestion
   apikey = DD_API_KEY
   appkey = DD_APP_KEY
```

File: content/en/developers/guide/dogshell.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+Dogshell uses a configuration file called `.dogrc` to store your API key, application key, and Datadog site.
+
+To configure Dogshell:
+1. Create a `.dogrc` file in your home directory:
+   {{< code-block lang="shell" >}}
+touch ~/.dogrc
+{{< /code-block >}}
+
+1. Add the following content to the file, replacing `DATADOG_API_KEY` and `YOUR_APPLICATION_KEY` with your API key and application key respectively:
```
Comment (hestonhoffman) [inline content/en/developers/guide/dogshell.md:31]:
```suggestion
1. Add the following content to the file, replacing `DD_API_KEY` and `DD_APP_KEY` with your API key and application key respectively:
```

File: content/en/developers/guide/dogshell.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+1. Create a `.dogrc` file in your home directory:
+   {{< code-block lang="shell" >}}
+touch ~/.dogrc
+{{< /code-block >}}
+
+1. Add the following content to the file, replacing `DD_API_KEY` and `DD_APP_KEY` with your API key and application key respectively:
+   ```conf
+   [Connection]
+   apikey = DD_API_KEY
+   appkey = DD_APP_KEY
```
Comment (neko-dd) [inline content/en/developers/guide/dogshell.md:35]: I think these should probably still have the angle brackets

**Review by neko-dd — APPROVED** [review]
One thing to change, otherwise LGTM

File: content/en/developers/guide/dogshell.md (hunk 6 above)
Comment (hestonhoffman) [reply content/en/developers/guide/dogshell.md]: I just realized I don't like `DD_` here cos that's usually a reserved environment variable. I'm going to use `MY_` instead - we use that fairly often. I'm not a huge fan of angle brackets cos I find it sometimes confuses users - they enclose the values in them by mistake.

**Pull request merged**